docker/Cypress.Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Sync this up with the Cypress version in package.json
```

Comment:
I tried to use the Cypress official image with included support and I don't know how to get it to work. May need to let it marinate a bit. For now I'm installing all the required deps from their docker image recommendation. `npm ci` fails in theirs due to a missing Python dep. 🤷🏼